TopMovies/View/TopMoviesVC/MovieCategoryTableViewCell.swift

```diff
@@ -126,6 +130,8 @@ extension MovieCategoryTableViewCell: UICollectionViewDelegate {
   func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
     if indexPath.item == props.movies.count {
       props.actionAllButton()
+    } else {
+      props.actionMovieDetail(props.movies[indexPath.item].movieId)
```

Comment:
That's not really nice, since it's not a guarantee that you will have a See all item as the last item, let's say you have only 1 movie.

Reply:
> That's not really nice, since it's not a guarantee that you will have a See all item as the last item, let's say you have only 1 movie.

I always have one more place for See all item button on movies.count + 1 position, so it will work even if i haven't movies at all